pwndbg/elf.py

```diff
@@ -68,6 +70,59 @@ def read(typ, address, blob=None):
     return obj
 
 
+@pwndbg.proc.OnlyWhenRunning
```

Comment:
Hmm the function seems it doesn't use anything that requires the program to be running, maybe we can drop that decorator from here?

Reply:
Sure, I have no opinion on that decorator. Just kept it because it was in the original code, I believe